docs-md/tutorials/clickstream-docker.md

```diff
@@ -166,7 +166,7 @@ Load the Streaming Data to KSQL
 1.  Launch the KSQL CLI:
 
     ```bash
-    docker-compose exec ksql-cli ksql http://ksql-server:8088
+    docker-compose exec ksql-cli ksql http://ksqldb-server:8088
```

Comment:
Should this be `ksqldb-cli`?

Reply:
Oh heck, yes!

Reply:
Jim is correct, the service is named `ksql-cli` in the docker-compose: https://github.com/confluentinc/examples/blob/5.3.1-post/clickstream/docker-compose.yml#L97 so that part is fine. However, `ksqldb-server:8088` should be `ksql-server:8088` so Mike has still found a bug! Will update accordingly.

Reply:
Ah, my bad. This example is using the CP Docker images, and the containers have the `ksql` name instead of `ksqldb`. Sorry for the confusion.